javascript/javascript1/week2/homework.js

```diff
@@ -0,0 +1,126 @@
+//Formal fullname
+
+
+function getFullname(firstname,surname,useFormalName = false){ 
+    if (useFormalName){         
+        return `Lord.${firstname} ${surname}`
+    }else{
```

Comment:
Would be nice to see logic implemented for using formal name but for a female. Eg. "Lady Someone Someone".